src/librustc_codegen_llvm/back/link.rs

```diff
@@ -218,15 +219,6 @@ fn preserve_objects_for_their_debuginfo(sess: &Session) -> bool {
     false
 }
 
-fn filename_for_metadata(sess: &Session, crate_name: &str, outputs: &OutputFilenames) -> PathBuf {
```

Comment:
Moved it so I can only depend on `librustc_codegen_utils` in the save analysis crate (see [here](https://github.com/rust-lang/rust/pull/54356/commits/a18ca5f9475d4b4afad6c561a9d4a28531bc6ce8))